src/stories/views/components/page/index/page_offline.hbs

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html 
+    class="no-js no-js-burgerMenu preload noMicroInteraction"
+    data-theme="hessenschau" 
+    lang="de"
+>
+    <head>
+
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+        <meta http-equiv="X-UA-Compatible" content="IE=Edge"/>
+        <title>hessenschau.de | Nachrichten aus Hessen</title>
+        <meta name="application-name" content="hessenschau.de" />
+        <meta name="copyright" content="Hessischer Rundfunk, Frankfurt, Germany" />
+        <meta name="author" content="hessenschau.de, Frankfurt, Germany" />
+        <meta name="publisher" content="hessenschau.de" />
+        <meta name="language" content="Deutsch" />
+
+    </head>
+
+    <body itemscope itemtype="http://schema.org/WebPage">
+        <div class="js-pageSwap">
+                      
+            {{> components/site_header/offline_header }}          
+            
+            <main 
+                onclick="void(0)" 
+                role="main" 
+                id="content"
+                class="flex items-center justify-center"
+            >
+                <div class="grid grid-page flex items-center flex-wrap justify-center">
+                    <div class="{{> components/grid/grid_group_wrapper_classes }}">
+                        <section class="grid items-center justify-center grid-cols-12 col-span-12 bg-white gap-x-6 gap-y-6">
+                            <div class="items-start relative col-span-12 flex gap-y-3 gap-x-4 flex-col">
+                                <div>
+                                    <div class="flex justify-center flex-wrap text-center">
+                                        {{> components/base/image/icon _icon="hilfe" _addClass="size-full h-52" _iconmap="404"~}}
+                                        </div>
+                                        <h3 class="text-lg sm480:text-2xl font-headingSerif dark:text-text-dark flex justify-center flex-wrap text-center mt-9">
+                                             {{loca "offline_page_info_text"}}
+                                        </h3>
+                                        <div class="flex justify-center flex-wrap text-center mt-9" onclick="window.location.reload();">
```

Comment:
the onclick needs to be placed on the button. You can do this with {{htmlProperties}} for the button